protocol/0082-ETHD-ethereum-data-source.md

```diff
@@ -111,80 +111,81 @@ Select {
 ### External Oracles - Creation
 
 1. Using the existing ways to create or update a market via governance proposals, define data sources for settlement and termination as the result of calling a read method of a smart contract on ethereum network. (Phase 2 of this step would be defining an oracle that is based on listening for events on ethereum network) (<a name="0082-ETHD-001" href="#0082-ETHD-001">0082-ETHD-001</a>)
```

Comment:
if this is a 2 phased AC should we split it and follow the suggested format in this [comment](https://github.com/vegaprotocol/specs/pull/1875/files#r1284248380)

Reply:
AC has been split and numbered correctly.